docs/references/release-checklist.md

```diff
@@ -11,8 +11,9 @@ Before we release a new version or beta version the following checklist should b
 * ✅ Manual and automated tests pass
   * See [Test Plan](#test-plan) below.
 * ✅ All translations have been updated
+  * String translations will need to be started manually from pre-release branch using [this](https://github.com/Azure/communication-ui-library/actions/workflows/run-td-build.yml) workflow. 
   * Ensure that strings for both components and composites have complete translations.
-  * If necessary, merge translations into `main` and cherry-pick updated translations to release branch.
+  * Once the string translations are completed they will need to be cherry picked into the release branch before publishing.
```

Comment:
Can we not run string translations directly in pre-release branch and create the release branch 5 days later so that the release branch has correct changes?

Reply:
Can't.

The prerelease branch is merged back into `main`. We should pretty much never merge string translations from a branch to another.